src/components/FeatureViews/AdminPortal/AdminDashboardCards.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
+import CardContent from "@material-ui/core/CardContent";
+import Avatar from "@material-ui/core/Avatar";
+import { stringToColor } from "../Accounts/accountUtils";
+
+const useStyles = makeStyles((theme) => ({
+  snapName: {
+    color: "#666666",
+    float: "left",
+    lineHeight: "1.1rem",
+    fontSize: "0.9375rem",
+    marginLeft: "21px",
+    marginTop: "12px",
+  },
+  number: {
+    float: "left",
+    marginLeft: "21px",
+    color: "#28ABD5",
+    fontSize: "2.8125rem",
+    fontWeight: "300",
+    fontStyle: "normal",
+  },
+  opCard: {
+    margin: "20px !important",
+    padding: "10px !important",
+  },
+  opAvatar: {
+    width: "65px",
+    height: "65px",
+    fontSize: "30px",
+    alignSelf: "center",
+    fontFamily: "Roboto",
+  },
+  opDetail: {
+    fontSize: "12px",
+    textAlign: "left",
+  },
+  amtdue: {
+    color: "red",
+  },
+}));
+
+export const Snapshot = (props) => {
+  const classes = useStyles();
+  return (
+    <Paper elevation={0}>
+      <Typography className={classes.snapName}>{props.snapName}</Typography>
+      <br />
+      <br />
+      <Typography className={classes.number}>{props.number}</Typography>
+    </Paper>
+  );
+};
+
+export const OutstandingPaymentCard = (op) => {
```

Comment:
don't abbreviate!!! write out the whole name to clarify what you mean. This is also the props for outstanding payment, so I'm not sure why you're calling all of the props as "op"

Reply:
you should also just destructure from the prop like `({initials, name, amtDue})` instead of `(op)`